addons/mrp/models/stock_rule.py

```diff
@@ -94,8 +95,8 @@ def _prepare_mo_vals(self, product_id, product_qty, product_uom, location_id, na
             'location_src_id': self.location_src_id.id or self.picking_type_id.default_location_src_id.id or location_id.id,
             'location_dest_id': location_id.id,
             'bom_id': bom.id,
-            'date_deadline': date_deadline,
-            'date_planned_start': date_deadline,
+            'date_deadline': fields.Datetime.to_string(date_deadline),
```

Comment:
not sure you need to keep fields.Datetime.to_string.
It's an old practice that has been fixed in the ORM